content/en/docs/ops/configuration/traffic-management/traffic-routing/index.md

```diff
@@ -0,0 +1,134 @@
+In general, there are three classes of protocols Istio understands:
+
+* HTTP, which includes HTTP/1.1, HTTP/2, and gRPC. Note that this does not include TLS encrypted traffic (HTTPS).
+* TLS, which includes HTTPS.
+* Raw TCP bytes.
+
+The [protocol selection](/docs/ops/configuration/traffic-management/protocol-selection/) document describes how Istio decides which protocol is used.
+
+The use of "TCP" can be confusing, as in other contexts it is used to distinguish between other L4 protocols, such as UDP.
+When referring to the TCP protocol in Istio, this typically means we are treating it as a raw stream of bytes,
+and not parsing application level protocols such as TLS or HTTP.
+
+## Traffic Routing
+
+When an Envoy proxy receives a request, it must decide where, if anywhere, to forward it to.
+By default, this will be to the original service that was requested, unless [customized](/docs/tasks/traffic-management/traffic-shifting/).
+How this works depends on the protocol used.
+
+### TCP
+
+When processing TCP traffic, Istio has a very small amount of useful information to route the connection - only the destination IP and Port.
+These attributes are used to determine the intended Service; the proxy is configured to listen on each service IP (`<Kubernetes ClusterIP>:<Port>`) pair and forward traffic to the upstream service.
+
+For customizations, a TCP `VirtualService` can be configured, which allows [matching on specific IPs and ports](/docs/reference/config/networking/virtual-service/#L4MatchAttributes) and routing it to different upstream services than requested.
+
+### TLS
+
+When processing TLS traffic, Istio has slightly more information available than raw TCP: we can inspect the [SNI](https://en.wikipedia.org/wiki/Server_Name_Indication) field presented during the TLS handshake.
+
+For standard Services, the same IP:Port matching is used as for raw TCP.
+However, for services that do not have a Service IP defined, such as [ExternalName services](#externalname-services), the SNI field will be used for routing.
+
+Additionally, custom routing can be configured with a TLS `VirtualService` to [match on SNI](/docs/reference/config/networking/virtual-service/#TLSMatchAttributes) and route requests to custom destinations.
+
+### HTTP
+
+HTTP allows much richer routing than TCP and TLS. With HTTP, you can route individual HTTP requests, rather than just connections.
+In addition, a [number of rich attributes](/docs/reference/config/networking/virtual-service/#HTTPMatchRequest) are available, such as host, path, headers, query parameters, etc.
+
+While TCP and TLS traffic generally behave the same with or without Istio (assuming no configuration has been applied to customize the routing), HTTP has significant differences.
+
+* Istio will load balance individual requests. In general, this is highly desirable, especially in scenarios with long-lived connections such as gRPC and HTTP/2, where connection level load balancing is ineffective.
```

Comment:
only when sticky session is not used (which is not enabled by default)?

Reply:
Regardless of if consistent hash is used, its still load balancing individual requests technically - they just go to the same place. I don't want to make this complex here as a general concept doc, the API docs discuss the behavior of this feature

Reply:
fair point - got it confused when you had long-lived connections.